src/features/pages/home/live-pricing/market-tabs/utils.tsx

```diff
@@ -15,7 +17,7 @@ export const market_buttons: TMarketButtons[] = [
         src: Forex,
         selected_src: SelectedForex,
         button_text: 'Forex',
-        market_name: 'forex',
+        market_name: 'fx',
```

Comment:
Sorry for asking some stupid questions, but why change the name instead of using the previous name? Is there a naming guidelines I can refer to?